src/CoreBundle/Resources/translations/admin.en.xliff

```diff
@@ -2202,6 +2202,10 @@
                 <source>chameleon_system_core.image_chooser.tree_node</source>
                 <target>Directory</target>
             </trans-unit>
+            <trans-unit id="chameleon_system_core.document_manager.folder">
```

Comment:
Please add a plural `s` at the end (also for the translated value).

Reply:
done